src/client/Table.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import data from './data.json';
+
+function TableItem(props){
+  const isEven = (props.seq % 2 === 0) ? true : false;
+  const className = isEven ? 'table__row table__row--even' : 'table__row table__row--odd';
+
+  return(
+    <tr className={className}>
+        <td>{props.seq}</td>
+        <td>{props.status}</td>
+        <td>{props.category}</td>
+        <td>{props.title}</td>
+        <td>{props.owner}</td>
+        <td>{props.priority}</td>
+      </tr>
+  )
+}
+
+function ShowTable(props){
+  const tableData = props.tableData;
+  const tableItems = tableData.map((item) => 
+    <TableItem key={item.seq}
```

Comment:
We may describe the name component more precisely. I suggest to rename to `<TableRow />` or `<Row />`